### ISSUE TITLE
Project Grows

### ISSUE THREAD

**Issue by toastyThink** [toastyThink/dog_park_finder#4]
Grows: 
	1) I would like to change the User Interface so that it does not block the built in Google maps features. Currently it is covering several at the top, any suggestions for a different UI layout would be great! 
	
2) Depending on how you enter the coordinate the app may break. I included if statements to catch the most common errors can that can occur. But it can still break, I am not sure how to fix this issue because it seems random at times.
	
3)  Any suggestions on adding new features would be appreciated!

4) Anything else you notice and think can be improved!

**Comment by Redd9292** [comment]
I agree that the app's interface needs improvement. There seems to be an issue with the map features, likely fixable with some CSS. Fixing coordinate errors should be a top priority since it's a crucial feature for users. Also, it would be great if users could turn on a GPS button to quickly find nearby dog parks.
